openedx/core/djangoapps/user_api/accounts/__init__.py

```diff
@@ -25,3 +26,13 @@
 
 # Indicates the user's preference that all their account information be private.
 PRIVATE_VISIBILITY = 'private'
+
+# Translators: This message is shown to users who attempt to create a new
+# account using an email address associated with an existing account.
+EMAIL_CONFLICT_MSG = u"It looks like {email_address} belongs to an existing account. " \
```

Comment:
I also planned on adding more strings here of the sort, to reuse these in tests (I've been through too many situations now where the error string is hardcoded into tests; it's a little annoying to have to deal with if it changes). But I'd still like to know if there's simply no harm here in translating before being used.